internal/service/encryptionatrest/resource_test.go

```diff
@@ -600,7 +665,9 @@ func testAccMongoDBAtlasEncryptionAtRestConfigAwsKms(projectID string, aws *admi
 				role_id              = "%s"
 			}
 		}
-	`, projectID, aws.GetEnabled(), aws.GetCustomerMasterKeyID(), aws.GetRegion(), aws.GetRoleId())
+
```

Comment:
[nit] tempting to refactor lines above and in the next function to use `%[1]q` instead of `%s` and similar to be consistent. WDYT?

Reply:
updated